.github/workflows/actions.yml

```diff
@@ -0,0 +1,84 @@
+name: CI
+
+on:
+  push:
+    branches: '**'
+    tags: 'v*'
+  pull_request:
+    branches: '**'
+
+env:
+  BUILD_VERSION: 23.6.0.${{ github.run_number }}
+
+jobs:
+  build-and-test:
+    runs-on: windows-2022
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup VSTest Path
+        uses: darenm/Setup-VSTest@v1.2
+
+      - name: Set up MSBuild
+        uses: microsoft/setup-msbuild@v1
+
+      - name: Restore NuGet packages
+        run: nuget restore
+
+      - name: Patch C++ Version Info
+        shell: pwsh
+        run: |
+          $content = Get-Content Profiler/version.h
+          $newContent = $content -replace '1\.0\.0\.0', "${env:BUILD_VERSION}"
+          $newContent = $newContent -replace '1,0,0,0', "${env:BUILD_VERSION.Replace('.', ',').Split('-')[0]}"
+          $newContent | Set-Content Profiler/version.h
+
+      - name: Build
+        run: msbuild Cqse.Teamscale.Profiler.Dotnet.sln /p:Platform=Win32 -property:Configuration=Release
```

Comment:
I think we patched only the version for the c++ assemblies, but not for the c# assembly infos in:

- UploadDaemon/Properties/AssemblyInfo.cs
- DumpPdb/Properties/AssemblyInfo.cs

This could be easily done for dotnet build with `dotnet build SolutionName.sln -c Release /property:Version=1.2.3.4`
However I think we cannot use dotnet build right now correct?

Reply:
Update, I think we can use this Github Action for that: https://github.com/secondbounce/assemblyinfo-update

Reply:
AFAIU `msbuild` might also be able to pass that property, I will try it out locally. 👍

Reply:
`msbuild` would need some custom scripting to work. I also tried to do the build with `dotnet build` locally but it ran into some weird issues. :grin:

Thanks for looking up the GitHub Action, it works great! 😊